src/Concise/Core/TestCase.php

```diff
@@ -276,10 +299,14 @@ protected function shouldAccessProperty($object, $property)
         method_exists($object, '__get');
     }
 
-    public function getProperty($object, $property)
+    public function getProperty($object, $property, $class = null)
```

Comment:
There needs to better docs for `getProperty()` and `setProperty` since they are part of the public API. I wouldn't expect someone to have to lookup the docs online.